app/src/main/java/com/yeonae/chamelezone/view/search/KeywordTabFragment.kt

```diff
@@ -49,47 +46,47 @@ class KeywordTabFragment : Fragment(), SearchContract.View {
         setAdapter()
 
         btn_cafe.setOnClickListener {
-            listener?.keywordSelected("카페")
+            listener.keywordSelected("카페")
```

Comment:
```suggestion
            listener.keywordSelected(Keyword.CAFE)
```
상수 혹은 enum class로 관리해주세요.

Reply:
이 부분 제가 서버에서 받아서 써야되는데 다음 pr에 수정해서 올리겠습니다